articles/cross-origin-authentication/index.md

```diff
@@ -45,7 +45,9 @@ Configuring your application for cross-origin authentication is a process that r
 There are some cases when third party cookies will not be available. Certain browser versions do not support third party cookies and, if they do, there will be times that they will be disabled in a user's settings. You can use **auth0.js** in your application on a dedicated page to properly handle cases when third-party cookies are disabled. **This page must be served over SSL**.
 
 ::: note
-Note that using `crossOriginVerification` as a fallback will only work if the browser is on the support matrix as **Yes** under "Third-Party Cookies Disabled". For some browsers, such as **Chrome** and **Opera** (Desktop & Android versions of both), when third party cookies are disabled, cross-origin authentication will not work at all unless you enable [Custom Domains](/custom-domains).
+Note that using `crossOriginVerification` as a fallback will only work if the browser is on the support matrix as **Yes** under "Third-Party Cookies Disabled". For some browsers, such as **Chrome**, **Opera** and **Safari¹** (Desktop & Mobile versions), when third party cookies are disabled, cross-origin authentication will not work at all unless you enable [Custom Domains](/custom-domains).
```

Comment:
Link to the support matrix?

Reply:
https://auth0.com/docs/cross-origin-authentication#browser-testing-matrix

Reply:
There's a link in a section above, I think it's enough.

![image](https://user-images.githubusercontent.com/941075/39953531-19dfb60a-5584-11e8-9c4a-c1b85dccbd7c.png)